rules/opentitan/hw.bzl

```
@@ -71,7 +128,13 @@ describe_top_rule = rule(
     doc = """Create a target that provides the description of a top in the form of an OpenTitanTopInfo provider.""",
     attrs = {
         "hjson": attr.label(mandatory = True, allow_single_file = True, doc = "toplevel hjson file generated by topgen"),
-        "ip_hjson": attr.string_keyed_label_dict(allow_files = True, doc = "mapping from hjson files to IP name"),
+        "ip_map": attr.string_keyed_label_dict(
+            allow_files = True,
+            # Transition to host becomes some of those attributes could targets (e.g. python libraries) that only work
+            # on the host platform.
```

nit: Seems like a few typos were here.

```suggestion
            # Transition to host because some of those attributes could use targets (e.g. python libraries) that only work
            # on the host platform.
```